tmo-monitor.py

```diff
@@ -134,79 +150,217 @@ def sha256url(self, val1, val2):
+      if tmp != None:
+        self.login[var] = tmp
+    for var in {'interface', 'ping_host', 'ping_count', 'ping_interval'}:
+      tmp = os.environ.get('tmo_' + var)
+      if tmp != None:
+        self.ping[var] = tmp  
+    for var in {'primary_band', 'secondary_band'}:
+      tmp = os.environ.get('tmo_' + var)
+      if tmp != None:
+        splits = tmp.split(',')
+        self.connection[var] = splits
+    tmp = os.environ.get('tmo_enbid')
+    if tmp != None:
+      self.connection['enbid'] = tmp
+    for var in {'uptime', 'ping', '4G_band', '5G_band', 'enbid'}:
+      tmp = os.environ.get('tmo_' + var + '_reboot')
+      if tmp != None:
+        if tmp.lower() == 'true':
+          self.reboot[var] = True
+        else:
+          self.reboot[var] = False
+    tmp = os.environ.get('tmo_skip_reboot')
+    if tmp != None:
+      self.skip_reboot = tmp
+    tmp = os.environ.get('tmo_print_config')
+    if tmp != None:
+        if tmp.lower() == 'true':
+          self.general['print_config'] = True
+        else:
+          self.general['print_config'] = False
+     
+    
+  def parse_commandline(self):
+    self.parser = argparse.ArgumentParser(description='Check T-Mobile Home Internet cellular band(s) and connectivity and reboot if necessary')
+    # login settings
+    self.parser.add_argument('username', type=str, help='the username (most likely "admin")', nargs='?')
+    self.parser.add_argument('password', type=str, help='the administrative password (will be requested at runtime if not passed as argument)', nargs='?')
+    # ping configuration 
+    self.parser.add_argument('-I', '--interface', type=str, help='the network interface to use for ping. pass the source IP on Windows')
+    self.parser.add_argument('-H', '--ping-host', type=str, default='google.com', help='the host to ping (defaults to google.com)')
+    self.parser.add_argument('--ping-count', type=int, default='1', help='how many ping health checks to perform')
+    self.parser.add_argument('--ping-interval', type=int, default='10', help='how long in seconds to wait between ping health checks')
+    #reboot settings
+    self.parser.add_argument('-R', '--reboot', action='store_true', help='skip health checks and immediately reboot gateway')
+    self.parser.add_argument('-r', '--skip-reboot', action='store_true', help='skip rebooting gateway')
+    self.parser.add_argument('--skip-bands', action='store_true', help='skip check for connected band')
+    self.parser.add_argument('--skip-5g-bands', action='store_true', help='skip check for connected 5g band')
+    self.parser.add_argument('--skip-ping', action='store_true', help='skip check for successful ping')
+    self.parser.add_argument('--skip-enbid', action='store_true', help='skip check for connected eNB ID')
+    self.parser.add_argument('--uptime', type=int, default=90, help='how long the gateway must be up before considering a reboot (defaults to 90 seconds)')
+    #connection configuration
```

Comment:
inconsistent comment spacing
```suggestion
    # connection configuration
```